specification/deploymentmanager/resource-manager/readme.nodejs.md

```diff
@@ -7,6 +7,7 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 nodejs:
   azure-arm: true
   package-name: azure-arm-deploymentmanager
+  package-version: 0.9.0

```

Comment:
Please remove this property. We no longer put package-versions in these readme files.